CRM/Eventinvitation/Queue/Runner/Job.php

```diff
@@ -150,8 +150,8 @@ protected function getTemplateTokens(int $participantId): array
         $link = '';
         if (
             is_array($settings)
-            && array_key_exists(CRM_Eventinvitation_Form_Settings::LINK_TARGET_IS_CUSTOM_FORM_NAME, $settings)
-            && array_key_exists(CRM_Eventinvitation_Form_Settings::CUSTOM_LINK_TARGET_FORM_NAME, $settings)
+            && ! empty($settings[CRM_Eventinvitation_Form_Settings::LINK_TARGET_IS_CUSTOM_FORM_NAME])
+            && ! empty($settings[CRM_Eventinvitation_Form_Settings::CUSTOM_LINK_TARGET_FORM_NAME])
```

Comment:
`empty()` should be avoided in favor of a type-safe comparison.

Reply:
replaced empty by custom-verification function in commit 2fb77f3ac1733aa1d9c7baf917c1b88da688d92b